ipaserver/plugins/cert.py

```diff
@@ -215,6 +216,11 @@ def caacl_check(principal_type, principal, ca, profile_id):
             )
         )
 
+def ca_kdc_check(ldap, basedn, hostname):
```

Comment:
You re-wrote the implementation but forgot to remove `basedn` param from signature, that's why you get pylint errors about no value for argument.